packages/docs-data/compile-docs-data

```diff
@@ -23,6 +24,8 @@ const DOCS_DATA_FILENAME = "docs.json";
 const DOCS_RELEASES_FILENAME = "releases.json";
 const DOCS_VERSIONS_FIELENAME = "versions.json";
 
+const PACKAGES = ["core", "datetime", "docs", "docs-app", "icons", "labs", "select", "table", "timezone"];
```

Comment:
why do you need to maintain this list? I liked the previous approach; don't want to maintain _yet another_ list of packages here